pandas/info.py

```diff
@@ -1,20 +0,0 @@
-"""
-pandas - a powerful data analysis and manipulation library for Python
-=====================================================================
-
-See http://pandas.pydata.org/ for full documentation. Otherwise, see the
```

Comment:
Good idea to remove this file. To have the same effect (people doing `pd?` see some info), you can put eg what is above this comment in `__init__.py` as module docstring. It's not that useful, but it does also no harm to have a minimal explanation.